src/main/java/com/google/devtools/starlark/BUILD

```diff
@@ -25,6 +26,22 @@ java_binary(
     ],
 )
 
+[
+    jmh_java_benchmarks(
```

Comment:
There's no need to create one build target per file in benchmarks/*. The Java code is 100% identical for all of them. Instead just pass the list of filenames to a single Java program and do a loop in Java.